src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -37,20 +37,38 @@ import {
 
 let nextUniqueId = 0;
 
+/**
+ * The `sky-radio-group` component organizes radio buttons into a group. It is required for radio
+ * buttons on Angular reactive forms, and we recommend using it with all radio buttons.
+ * On Angular forms, the component manages the selected values and keeps the forms up-to-date.
+ * When users select a radio button, its value is driven through an `ngModel` attribute that you specify on the `sky-radio-group` element.
+ */
 @Component({
   selector: 'sky-radio-group',
   templateUrl: './radio-group.component.html'
 })
 export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit, OnDestroy {
 
+/**
+ * Sets the radio button group's `aria-labelledby` attribute to support
+ * [accessibility](https://developer.blackbaud.com/skyux/components/radio#accessibility). The value
+ * should be the HTML element ID (without the leading #) of the element that labels the radio
```

Comment:
I'm pretty sure `#` should be a code snippet. I checked the existing docs, and it's a code snippet in modal and flyout but not checkout and radio button. I think that's an oversight on the latter 2 components and that we should make it a code snippet here, but it can't hurt to check with Alex or another dev.

Reply:
Looks like this wasn't addressed. Not a huge deal, but just pointing it out so that we can either address it or resolve this comment thread as "will not address." Do you want me to follow up with Alex to determine whether # should be a code snippet since we have it both ways in the existing docs?

Reply:
Made `#` a code snippet